--- a/iwinfo.c
+++ b/iwinfo.c
@@ -246,7 +246,7 @@ rpc_iwinfo_info(struct ubus_context *ctx
 
 	blob_buf_init(&buf, 0);
 
-	rpc_iwinfo_call_str("phy", iw->phyname);
+	//rpc_iwinfo_call_str("phy", iw->phyname);
 
 	rpc_iwinfo_call_str("ssid", iw->ssid);
 	rpc_iwinfo_call_str("bssid", iw->bssid);
--- a/session.c
+++ b/session.c
@@ -994,7 +994,7 @@ rpc_login_setup_acl_file(struct rpc_sess
 
 	blob_buf_init(&acl, 0);
 
-	if (!blobmsg_add_json_from_file(&acl, path)) {
+	if (!blobmsg_add_json_from_file(&acl, (char *)path)) {
 		fprintf(stderr, "Failed to parse %s\n", path);
 		goto out;
 	}
